### PR TITLE
Add the initial version of "Annotating Chrismas Trees"

### LINKED ISSUE #437: Annotating Christmas Trees
Which Perl module do you want to write about:

Treex::PML

Please give a short description of the article you want to write. Two to three sentences is fine:

I want to present a module that propels the tree editor TrEd. I talked about TrEd in my TPRC 2024 talk, but unlike there, I'd like to only present the basics in the article.

### PR DESCRIPTION
**Pull request by choroba**
Fixes #437.

### CODE REVIEW

**Comment by oalders** [comment]
Thank you very much for your contribution. I have unfortunately come up against some time constraints and I don't have the resources to dedicate to another 12 or so proper article reviews. If we don't get a review in, your article will still get merged, but I'll likely only have time to look for typos and ensure formatting is mostly correct.

If you know of a peer who would be willing to review your work, I'd encourage you to ask them to leave a review on this pull request. If you find yourself with any extra time, feel free to self-assign any outstanding Advent pull request(s) and leave a helpful review. [The editing guidelines](https://github.com/perladvent/Perl-Advent/blob/main/EDITING.md) should be helpful in this regard.

Basically the situation this year is that most articles arrived in late November/early December, meaning that there is now a bottleneck at this time of year which is busy for many of us. This is not a judgement on anyone -- all articles are welcome and I always expect some to come in later in the game. In fact, there are still more articles to come which do not yet have pull requests.

In order to ensure that everyone's content gets a fair chance for review, I'm looking to cast a wider net. Hopefully spreading the effort around this way will help us continue the tradition of high value Advent articles in 2025.

File: 2024/incoming/annot-trees.pod [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+“Yes, but…” spluttered the elf.
+
+“Let’s meet after the lunch and see how we can share the knowledge,” commanded
+Santa.
+
+=head2 There’s trees and there’s trees
+
+In the afternoon (by the way, it was already dark, since it was the North Pole
+and summer was over) Santa met with the COO and CCO (where the second letter
+stands for Ornaments or Cards, respectively).
```
Comment (JJ) [inline 2024/incoming/annot-trees.pod:32]: Maybe add the CBO, in charge of baubles?

File: 2024/incoming/annot-trees.pod [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+
+“Can you show us what your department uses to visualise trees?” asked Santa,
```
Comment (JJ) [inline 2024/incoming/annot-trees.pod:34]: Turning to British spelling, are we?

**Review by JJ — APPROVED** [review]
Even if the tool is actually written in Perl, I'd like to see a bit of more Perl here... As in code.

File: 2024/incoming/annot-trees.pod [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+but it’s not the sole business they’re into. In fact, they’ve recently engaged
+in production of Christmas ornaments and Christmas cards, too.
+
+As is common, starting a new business brings new problems. The elves needed a
+way to design decorated Christmas trees to evaluate their ornament proposals
+and their combinations. They started by planting small trees and decorating
+them by hand, but quickly found out this approach didn’t scale.
+
+At a C-level meeting, Santa listened to laments of the managing elf
+responsible for the ornaments and narrowed his eyes to another young elf at the other corner of the table.
```
[the remark this answers is not in the document]
Comment (openstrike) [reply 2024/incoming/annot-trees.pod:19]: I think "narrowed his eyes" is fine. But I would follow it with "at" rather than "to".

**Comment by oalders** [comment]
@choroba do you know when you'll be able to look at the review comments?

File: 2024/incoming/annot-trees.pod [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+
+“You had something for arranging trees, right?” asked Santa.
+
+“Yes, but…” spluttered the elf.
+
+“Let’s meet after the lunch and see how we can share the knowledge,” commanded
+Santa.
+
+=head2 There’s trees and there’s trees
```
[the remark this answers is not in the document]
Comment (choroba) [reply 2024/incoming/annot-trees.pod:28]: A native speaker needed to judge. I've read this expression somewhere (it mentioned "cars", so plural), but I have no idea how common it is.

File: 2024/incoming/annot-trees.pod (hunk 2 above)
Comment (choroba) [reply 2024/incoming/annot-trees.pod:34]: I think I keep it through the whole article 🇬🇧

File: 2024/incoming/annot-trees.pod [hunk 5]
```diff
@@ -0,0 +1,167 @@
+Author: E. Choroba
+Title: Annotating Christmas Trees
+Topic: Natural Language Processing, Graph Visualisation
+
+=encoding utf8
+
+=head2 Lesser-known businesses
+
+Everybody knows that the North Pole™ produces and delivers Christmas gifts,
+but it’s not the sole business they’re into. In fact, they’ve recently engaged
```
[the remark this answers is not in the document]
Comment (choroba) [reply 2024/incoming/annot-trees.pod:10]: I don't think the article is a formal writing.

**Comment by choroba** [comment]
> @choroba do you know when you'll be able to look at the review comments?

I tried to resolve most of them, see above.

**Comment by oalders** [comment]
Thanks, @choroba and @JJ!
